BenchBnb/BenchBnb/FormModels/CreateReview.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace BenchBnb.FormModels
+{
+    public class CreateReview
+    {
+        public int Id { get; set; }
```

Comment:
Is this supposed to represent the ID of the review or the ID of the bench that the review is being added to?

Reply:
The Id of the reviewID

Reply:
Go ahead and remove this property. You shouldn't need an `Id` property on a "create" form model.